Source/diablo.cpp

```diff
@@ -7,6 +7,17 @@
 #include <cstdint>
 #include <string_view>
 
+#ifdef USE_SDL3
+#include <SDL3/SDL_events.h>
+#include <SDL3/SDL_keycode.h>
+#else
+#include <SDL.h>
+
+#ifdef USE_SDL1
+#include "utils/sdl2_to_1_2_backports.h"
+#endif
+#endif

```

Comment:
Do you think in the long run we should stick with this, or something like:

```cpp
#if defined(USE_SDL1)
#include "utils/sdl3_to_2_backports.h"
#elif defined(USE_SDL2)
#include "utils/sdl3_to_1_2_backports.h"
#else
#include <SDL.h>
#endif
```

Reply:
Or maybe fully abstract it similar to how you are doing it in Source/utils/endian_swap.hpp

Reply:
I hope that in the long run we can remove SDL2 support so we'll only have SDL3, SDL1, and sdl3_to_1_2_backports.h